### DISCUSSION TITLE
How to set the component size global

### DISCUSSION THREAD

**Discussion by yansenlei** [quasarframework/quasar#13923] [General - Components / Directives / etc]
### Description

```js
app.use(ElementPlus, { size: 'small', zIndex: 3000 })
```
Similar options to Element-Plus, Maybe we can do this:
```js
app.use(Quasar, {
  size: 'xs'
})
```

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

**Comment by yansenlei** [comment]
@rstoenescu sir, Do you have any plans for this idea?

**Comment by jamesRUS52** [comment]
up
